widgetry/src/screen_geom.rs

```diff
@@ -97,6 +97,7 @@ impl ScreenRectangle {
     }
 }
 
+// REVIEW: Rename to something shorter? e.g. Dims / Size
```

Comment:
I was trying to emphasize that these're measuring screen-space. But that distinction has become way less important; I think `Dims` would be fine.